src/i18n/en.json

```diff
@@ -834,14 +834,18 @@
         "sh": "Schleswig-Holstein",
         "thue": "Thuringia",
         "ev": "Evangelical church",
+        "ev-q": "Which church or Diakonisches Werk does your request concern?",
```

Comment:
I guess you didn't find a good translation here. What about just using "Diaconia" like [Wikipedia](https://en.wikipedia.org/wiki/Diaconia)?

Reply:
I'm not well-versed with this, but I think "Diaconia" describes the theological concept rather than the institution.